package/yast2-storage.changes

```diff
@@ -1,30 +1,73 @@
 -------------------------------------------------------------------
+Mon Aug 21 10:32:52 UTC 2017 - igonzalezsosa@suse.com
+
+- Fix Btrfs default subvolume name detection (bsc#1044434 and
+  bsc#1044250)
+- Read default subvolume name for Btrfs filesystems when using
+  the expert partitioner (bsc#1040154)
+
+-------------------------------------------------------------------
+Wed Aug 16 11:34:48 CEST 2017 - shundhammer@suse.de
+
+- Make filesystem type for home and root configurable in control.xml
+  (bsc#1051762)
+
+-------------------------------------------------------------------
 Thu Jul 20 10:45:18 CEST 2017 - aschnell@suse.com
 
 - fixed check whether snapshots are proposed (bsc#1049108)
 - 3.1.109
 
 -------------------------------------------------------------------
+Mon Jun 26 11:13:00 CEST 2017 - shundhammer@suse.de
+
+- Allow different mount point for home partition (Fate#323532)
+- 3.1.108.2
+
+-------------------------------------------------------------------
 Wed Mar 22 08:45:22 UTC 2017 - mfilka@suse.com
 
 - bnc#1026027
   - dropped /etc/init.d/boot.crypto* invocations. Since these SysV
     scripts were replaced by native systemd implementation already.
-- 3.1.108 
+- 3.1.108.1 
 
 -------------------------------------------------------------------
 Mon Mar 20 09:12:25 UTC 2017 - mfilka@suse.com
 
 - bnc#1026027
   - dropped calls to /sbin/insserv
-- 3.1.107
+- 3.1.107.1
```

Comment:
No one will remove them, but since now it will be the CASP and SP2 branch for maintenance updates so the changelog will be wrong for one of them being strict with the versions released